google/resource_compute_instance.go

```diff
@@ -488,6 +488,33 @@ func resourceComputeInstance() *schema.Resource {
 				},
 			},
 
+			"shielded_vm_config": {
+				Type:     schema.TypeList,
```

Comment:
I made this as `TypeList` with `MaxItems: 1` like some other configuration options for this resource. This would make the config a block rather than a map as suggested in the original issue description. I wasn't sure which was preferable here.

Reply:
Terraform maps are only string -> string, and appropriate for user-modifiable KV pairs like `labels`. Since this is representing a nested object, this is the correct Terraform schema.

Reply:
got it, makes sense!